File: BookStore.DataAccess/Models/UesrsFilterModel/UsersFilterModel.cs
using BookStore.DataAccess.Models.Base;
using static BookStore.DataAccess.Models.Enums.Enums;

namespace BookStore.DataAccess.Models.UesrsFilterModel
{
    public class UsersFilterModel : BaseFilterModel
    {
        public UserSortType SortType { get; set; }
        public UserStatus UserStatus { get; set; }
    }
}

File: StoreClient1/src/app/shared/models/Users/users-filter-model.ts
import {UserSortType} from 'src/app/shared/enums/user-sort-type';
import {UserStatus} from 'src/app/shared/enums/user-status';
import {BaseFilterModel} from '../Base/base-filter-model';

export class UsersFilterModel extends BaseFilterModel {
  sortType: UserSortType;
  userStatus: UserStatus;
}
